### PR TITLE
docs(site): Fix the error of the FAQ reference link on the site

### PR DESCRIPTION
**Pull request by rxliuli**
ref: https://g6.antv.vision/zh/docs/manual/FAQ/faq

##### Checklist

- [ ] `npm test` passes
- [ ] tests and/or benchmarks are included
- [x] commit message follows commit guidelines

##### Description of change

image

---

It seems that another submission is also automatically merged into this PR. . . just fixed some type definition errors

### CODE REVIEW

File: packages/plugin/src/bundling/index.ts [hunk 1]
```diff
@@ -60,6 +60,9 @@ function projectPointToEdge(p: Point, e: VectorPosition): Point {
 }
 
 export default class Bundling extends Base {
+  constructor(config?: BundlingConfig) {
+    super(config);
+  }
```
Comment (Yanyan-Wang) [inline packages/plugin/src/bundling/index.ts:65]: 诶？不是改官网链接问题么，为啥这里会有修改？

Comment (rxliuli) [reply packages/plugin/src/bundling/index.ts:65]: 额，应该是另一个提交自动合并到这个 PR 了（就是修复一些类型定义的问题）

Comment (rxliuli) [reply packages/plugin/src/bundling/index.ts]: ok，吾辈重新推了一下，你合并这个 PR 之后我再添加吧

Comment (Yanyan-Wang) [reply packages/plugin/src/bundling/index.ts]: 感谢感谢

**Review by Yanyan-Wang — APPROVED** [review]

**Pull request merged**